src/components/Weapons.js

```diff
@@ -68,8 +95,76 @@ const Weapons = ({
           <IconButton onClick={addWeapon}>
             <i className="material-icons">done</i>
           </IconButton>
+           <Row>
+           <h2>Can&apos;t find your weapon?</h2>
+           <Button variant="contained" color="primary" onClick={() => setAddingUnique(true)}>Add Unique Weapon</Button>
+           </Row>
            </Row>
       }
+      {
+        isAddingUnique
+        && <>
+        <Row>
+          <h3>Add Unique Weapon</h3>
+          <Spacer />
+        </Row>
+          <Row>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
+                  style={{ width: '6rem' }}
+                  variant="outlined"
+                  label="Name"
+                  onChange={handleValueChange('name')}
+                />
+          </BasicBox>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
+                  style={{ width: '6rem' }}
+                  variant="outlined"
+                  label="Category"
+                  onChange={handleValueChange('category')}
+                />
+          </BasicBox>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
+                  style={{ width: '6rem' }}
+                  variant="outlined"
+                  label="Damage Dice"
+                  onChange={handleValueChange('damage_dice')}
+                />
+          </BasicBox>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
+                  style={{ width: '6rem' }}
+                  variant="outlined"
+                  label="Damage Type"
+                  onChange={handleValueChange('damage_type')}
+                />
+          </BasicBox>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
+                  style={{ width: '6rem' }}
+                  variant="outlined"
+                  label="Weight"
+                  onChange={handleValueChange('weight')}
+                />
+          </BasicBox>
+          <BasicBox style={{ margin: '.75rem' }}>
+                <TextField
```

Comment:
This may work best as a set of check boxes so they can select multiple pre-defined properties.